projects/ion/src/lib/message/message.component.ts

```diff
@@ -0,0 +1,44 @@
+import { IconType } from './../icon/icon.component';
+import { Component, Input, OnInit } from '@angular/core';
+
+export interface IonMessageProps {
+  label: string;
+  type?: Statustype;
+  iconType?: IconType;
+}
+
+export type Statustype =
```

Comment:
```suggestion
export type MessageStatusType =
```

change this name to not confuse with status of alert and notification.